### PR TITLE
Improve README.md a little bit.

### PR DESCRIPTION
**Pull request by lensvol**
I've decided to polish README.md a bit in attempt to pacify my OCD. 

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
...
 
-We have several primary objectives with this linter:
+We have several primary objectives with regards to this linter:
 
-0. Enforce `python3.6+` usage
-1. Significantly reduce code's complexity and make it more maintainable
-2. Enforce "There should be one-- and preferably only one --obvious way to do it" rule
+0. Enforce usage Python 3.6 or higher
+1. Significantly reduce complexity of your code and make it more maintainable
+2. Enforce "There should be one --and preferably only one-- obvious way to do it" rule
```
Comment (sobolevn) [inline README.md]: The exact quote is: `There should be one-- and preferably only one --obvious way to do it.`

File: README.md [hunk 2]
```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
 
 ## What we are about
 
-We have several primary objectives with this linter:
+We have several primary objectives with regards to this linter:
 
-0. Enforce `python3.6+` usage
-1. Significantly reduce code's complexity and make it more maintainable
-2. Enforce "There should be one-- and preferably only one --obvious way to do it" rule
+0. Enforce usage Python 3.6 or higher
```
Comment (sobolevn) [inline README.md]: Can you please revert this specific line?

File: README.md [hunk 3]
```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
...
+2. Enforce "There should be one --and preferably only one-- obvious way to do it" rule
 3. Create consistent coding and naming style
 
 You can find all error codes and plugins [in the docs](https://wemake-python-styleguide.readthedocs.io/en/latest/_pages/violations/index.html).
 
 
 ## What we are not
 
-We are here *not* to:
+We are *not* here to:
```
Comment (sobolevn) [inline README.md]: Are you sure that this is a correct word order? 
Meaning: "what topics are not our objectives".

Comment (sobolevn) [reply README.md]: So, maybe we can rephrase it?

File: README.md [hunk 4]
```diff
@@ -56,24 +56,24 @@ in the docs for examples and integrations.
 
 ## What we are about
 
-We have several primary objectives with this linter:
+We have several primary objectives with regards to this linter:
```
Comment (sobolevn) [inline README.md]: That's not about this linter. But that's about the code that will be produces with the help of this linter.

Comment (sobolevn) [reply README.md]: So, it might be just `We have several primary objectives`

**Comment by sobolevn** [comment]
@lensvol can you please also fix this typo: `See "Contributing" section file in the docs`. 
Should be: `See "Contributing" section in the docs`

File: README.md (hunk 4 above)
Comment (lensvol) [reply README.md]: Yeah, I was not sure in regards to what you have those objectives. Cool, will fix it :)

**Comment by sobolevn** [comment]
@lensvol awesome! Can you please address my other comments and I will be happy to get this merged.

Btw, we have met in person at least once. There was a python event in Yandex almost 3 or 4 years ago. You had a talk about RedBaron. I have asked you something about RedBaron and Pycharm integration afterwards. 🙂

File: README.md (hunk 2 above)
Comment (lensvol) [reply README.md]: Okay, will do :)

File: README.md (hunk 3 above)
Comment (sobolevn) [reply README.md]: 👍

**Pull request merged**